src/Sulu/Bundle/Sales/CoreBundle/Item/ItemManager.php

```diff
@@ -218,6 +218,15 @@ public function save(
 
         $item->setType($this->getProperty($data, 'type', BaseItem::TYPE_PRODUCT));
 
+        // Check if the product or addon relation on a saved still exists, else set type custom
```

Comment:
Check if the product or addon relation on a saved **item** still exists, else set type custom.